### PR TITLE
Add documentation release notes 2.6

### PR DESCRIPTION
**Pull request by kolchfa-aws**
Add documentation release notes 2.6

### Checklist
- [x ] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by natebower — APPROVED** [review]
LGTM

File: release-notes/opensearch-documentation-release-notes-2.6.0.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# OpenSearch Documentation Website 2.6.0 Release Notes
+
+The OpenSearch 2.6.0 documentation includes the following additions and updates.
+
+## New documentation for 2.6.0
+
+- Add changes to remote backed storage documentation for 2.6 release [#3098](https://github.com/opensearch-project/documentation-website/pull/3098)
+- Add documentation for log type updates in Security Analytics - pt. 2 [#3083](https://github.com/opensearch-project/documentation-website/pull/3083)
+- Add documentation for log type updates in Security Analytics [#3066](https://github.com/opensearch-project/documentation-website/pull/3066)
```
Comment (cwillum) [inline release-notes/opensearch-documentation-release-notes-2.6.0.md:9]: Not sure if it's really necessary. But could this one go above the "pt. 2" PR - 3083?

**Review by cwillum — APPROVED** [review]
LGTM

**Review by hdhalter — COMMENTED** [review]
I think we should only include one security analytics pr

**Pull request merged**